### PR TITLE
Coverage with CodeCov.io

### PR DESCRIPTION
**Pull request by roger-russel**
This pull request will integrate Coverage of Tests and it will be displayed at https://codecov.io/gh/roger-russel/beholder-web-client.

I put a badget into Readme.md, but it will only show the coverage of code after this PR be merged together with master.

### CODE REVIEW

**Comment by roger-russel** [comment]
This "No report found to compare against" should not be an error because this is the first commit to master with Codecov that's why it doesn't have something to compare.

File: Dockerfile [hunk 1]
```diff
@@ -16,11 +16,15 @@ RUN apt-get update && apt-get install -y \
   libpq-dev \
   build-essential \
   whois \
+  zlib1g-dev \
   && rm -rf /var/lib/apt/lists/*
 
-RUN pecl install yaml-1.3.1
+RUN yes '' | pecl install yaml-1.3.1
```
Comment (alanyoshida) [inline Dockerfile]: Just asking, what that `yes ' ' | ` does?

**Review by alanyoshida — APPROVED** [review]
Very cool man !

File: Dockerfile (hunk 1 above)
Comment (roger-russel) [reply Dockerfile]: "pecl install yaml" started to ask about a custom libname or "yes" to default.
pecl doesn't have an "-y" option, so I used the "yes" command, this "yes ' ' | " is a look foward combo command ( I beleave that it have another name, but I don't know. ) ,  it send an "yes" to every terminal ask made by the next command.

Comment (alanyoshida) [reply Dockerfile]: Man pages says:

       yes - output a string repeatedly until killed

SYNOPSIS
       yes [STRING]...
       yes OPTION

DESCRIPTION
       Repeatedly output a line with all specified STRING(s), or 'y'.

       --help display this help and exit

       --version
              output version information and exit

**Pull request merged**